tests-web/www/js/layers.js

```diff
@@ -1,49 +1,70 @@
 export const makePlainGeoJsonLayer = (text) => {
-  return new L.geoJSON(JSON.parse(text), { style: styleGeoJson });
-};
+  const intersectionColours = {
+    undefined: "#666", // for default tarmac
+    Connection: "#666",
+    MultiConnection: "#669",
+    Merge: "#969",
+    Crossing: "#966",
+    Terminus: "#999",
+    MapEdge: "#696",
+  };
 
-const intersectionColours = {
-  undefined: "#666", // for default tarmac
-  Connection: "#666",
-  MultiConnection: "#669",
-  Merge: "#969",
-  Crossing: "#966",
-  Terminus: "#999",
-  MapEdge: "#696",
+  return new L.geoJSON(JSON.parse(text), {
+    style: function (feature) {
+      if (feature.geometry.type === "Polygon") {
+        return {
+          color: intersectionColours[feature.properties?.complexity],
+          weight: 1,
+          fillOpacity: 0.7,
+        };
+      }
+      return { color: "#f55" };
+    },
+  });
 };
 
-const styleGeoJson = (feature) => {
-  if (feature.geometry.type === "Polygon") {
-    return {
-      color: intersectionColours[feature.properties?.complexity],
-      weight: 1,
-      fillOpacity: 0.7,
-    };
-  }
-  return { color: "#f55" };
+export const makeLanePolygonLayer = (text) => {
+  // These could change per locale
```

Comment:
I just lifted some colors from A/B Street's "classic" / original scheme. They're North American centric. It'd be pretty easy at this layer to just look at the locale or country code from Overpass and switch between some styles!